apollo/integrations/db/salesforce_crm_proxy_client.py

```diff
@@ -39,6 +39,43 @@ def execute(self, query: str) -> Dict:
 
         return {"description": description, "records": records, "rowcount": rowcount}
 
+    def execute_count_query(self, query: str) -> Dict:
+        headers = {
+            "Sforce-Query-Options": "batchSize=200",
```

Comment:
Nit: Why 200? Can we leave a comment, even if the answer is "it's an arbitrary value"?

Reply:
Done in 4788059bcd582edf9e186d4cf00be869548ad0ce.